src/OpenTelemetry/Trace/TracerSdk.cs

```diff
@@ -79,16 +79,16 @@ public override TelemetrySpan StartRootSpan(string operationName, SpanKind kind,
         }
 
         /// <inheritdoc/>
-        public override TelemetrySpan StartSpan(string operationName, TelemetrySpan parent, SpanKind kind, SpanCreationOptions options)
+        public override TelemetrySpan StartSpan(string operationName, TelemetrySpan parentSpan, SpanKind kind, SpanCreationOptions options)
```

Comment:
minor renaming for better readability - this not at all related to the PR. Hope its okay.